mod/01_core/set/all/fetch.rb

```diff
@@ -54,8 +54,11 @@ def fetch mark, opts={}
     if card.new_card?
       if opts[:new]
         return card.renew(opts) if !clean_cache_opts? opts
+      elsif opts[:skip_virtual]
```

Comment:
A card can be one of three things: unknown, virtual, or real.
known is, of course, not unknown(virtual or real).  new is not real (unknown or virtual)

[] is effectively shorthand for "fetch real" (it skips virtual).

And yes: #known? must return true if virtual and must therefore include modules for unknown cards, but at least it skips them for real cards.

As far as your last question: by default, fetch only returns known cards.  if it's unknown, it returns nil.  So you're describing the default case.

If you want a new card returned, you have to add a :new arg.  (eg `fetch mycardname, :new=>{}`). I could imagine further shorthand for that.  It does turn up with an empty hash often. I believe that as of yet there is no Card.get... Perhaps this:
````
def get mark, args={}
  fetch mark, :new=>args
end
````